atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java

```diff
@@ -37,7 +40,15 @@ private TransactionConstants() {
     public static final String COMMIT_TS_COLUMN_STRING = "t";
     public static final byte[] COMMIT_TS_COLUMN = PtBytes.toBytes(COMMIT_TS_COLUMN_STRING);
     public static final long FAILED_COMMIT_TS = -1L;
+    public static final byte[] DIRECT_ENCODING_ABORTED_TRANSACTION_VALUE = getValueForTimestamp(FAILED_COMMIT_TS);
+    public static final byte[] TICKETS_ENCODING_ABORTED_TRANSACTION_VALUE = PtBytes.EMPTY_BYTE_ARRAY;
+
+    public static final TransactionStatus ABORTED = TransactionStatuses.aborted();
+    public static final TransactionStatus IN_PROGRESS = TransactionStatuses.inProgress();
+    public static final TransactionStatus UNKNOWN = TransactionStatuses.unknown();
     public static final long LOWEST_POSSIBLE_START_TS = 1L;
+    public static final TransactionStatus PRE_START_COMMITTED =
+            TransactionStatuses.committed(AtlasDbConstants.STARTING_TS - 1);
```

Comment:
while we're here: what's the difference between `LOWEST_POSSIBLE_START_TS` and `AtlasDbConstants.STARTING_TS`? These seem linked, so might be good to unify them

Reply:
that is a good idea